build.gradle.kts

```diff
@@ -20,7 +20,6 @@ subprojects {
         plugin<org.jetbrains.kotlin.gradle.plugin.KotlinPluginWrapper>()
         plugin<JacocoPlugin>()
         plugin<io.gitlab.arturbosch.detekt.DetektPlugin>()
-        plugin<MavenPublishPlugin>()
```

Comment:
By removing the plugin by default and including the lines below, I'm avoiding publishing the testing artifacts (since those shouldn't be useful outside of the library)